src/components/MyNyplButton/MyNyplButton.js

```diff
@@ -105,23 +111,27 @@ class MyNyplButton extends React.Component {
   renderMyNyplButton() {
     let buttonClass = '';
     let iconClass = 'nypl-icon-wedge-down';
-    const icon = (<span className={`${iconClass} icon`} style={styles.MyNyplIcon}></span>);
+    let myNyplButtonLabel = (this.props.patronName) ? 'My Account' : 'Log In';
     const labelColorClass = (this.props.isLoggedIn) ? ' loggedIn' : '';
-    const myNyplButtonLabel = (this.props.patronName) ? 'My Account' : 'Log In';
     const loggedInFadeInAnimation = (this.props.patronName) ? ' animated fadeIn' : '';
 
     if (this.state.visible) {
       buttonClass = 'active';
       iconClass = 'nypl-icon-solo-x';
+      myNyplButtonLabel = 'Close';
     }
 
+    const icon = (<span className={`${iconClass} icon`} style={styles.MyNyplIcon}></span>);
+
     return (
       <a
         className={`MyNyplButton ${buttonClass}${labelColorClass}${loggedInFadeInAnimation}`}
         onClick={this.handleClick}
         style={_extend(styles.MyNyplButton, this.props.style)}
         href={this.props.target}
         role="button"
+        aria-haspopup="true"
+        aria-expanded={this.state.visible ? true : null}
```

Comment:
This should not be `false`. According to https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-actions-active-descendant.html, 

> The aria-expanded attribute is removed when the menu is closed.